smarty/sampler_smirky.py

```diff
@@ -835,7 +845,7 @@ def sample_types(self):
         env = random.choice(proposed_envList)
 
         # determine create or destroy:
-        if random.random() < 0.5:
+        if random.random() < 0.2:
```

Comment:
I will continue to think about the "right" way to do this. I think in the long run there should be some way that is based on data instead of being completely randomly, "so do we need to create or remove a parameter to better fit this data?". It also seems like if there are only initial types then it should know that removing isn't allowed so we don't waste time trying things that are always not allowed?